SPIFlashLogger.class.nut

```diff
@@ -224,6 +224,68 @@ class SPIFlashLogger {
         });
     }
 
+    function readSync(index = 1) {
```

Comment:
@deldrid1, do you really need the readSync itself or is it only used for first()/last() and can be made private?